src/TodoInput.jsx

```diff
@@ -0,0 +1,8 @@
+export default function TodoInput({ todo, onChange, onClick }) {
+  return (
+    <div>
+      <input type="text" value={todo} placeholder="할 일을 입력해 주세요" onChange={onChange} />
```

Comment:
```suggestion
      <input 
        type="text"
        value={todo} 
        placeholder="할 일을 입력해 주세요" 
        onChange={onChange} 
      />
```

어떤게 더 코드를 읽기 나은 것 같나요?
저는 속성이 2개 이상일 때는 줄바꿈해주면 코드가독성이 더 좋다고 생각하고 코드를 정리해주는 편이에요.
코드 가독성을 고려한 자기만의 코드 정리 기준을 세우시면 좋아요.

Reply:
프로젝트 내에 모든 element 들에 대해서 2개 이상의 속성일 때 개행을 하도록 수정했습니다!